src/components/EntityInfo.vue

```diff
@@ -5,30 +5,44 @@
         <v-col class="flex-column d-flex justify-space-between" cols="12" md="9">
           <header>
             <!-- Entity Name / IA Title -->
-            <div v-if="businessId" id="entity-legal-name" aria-label="Business Legal Name">
+            <div v-if="!!businessId" id="entity-legal-name" aria-label="Business Legal Name">
               <span>{{ getLegalName || 'Unknown Name' }}</span>
             </div>
-            <div v-if="tempRegNumber" id="incorp-app-title" aria-label="Incorporation Application Title">
-              <span>{{ getLegalName || getCorpTypeNumberedDescription(getLegalType)}}</span>
+            <div v-if="!!tempRegNumber" id="incorp-app-title" aria-label="Incorporation Application Title">
+              <span>{{ getLegalName || GetCorpNumberedDescription(getLegalType)}}</span>
```

Comment:
What do you think of taking the opportunity to refactor the EntityInfo component?  It looks like we might be mixing a number of different display elements:
- entity name, 
- entity status, 
- entity header menu, and; 
- entity key facts

Reply:
A single component is fine until it either gets too big or too complicated.  In this case, I was thinking that splitting the component into separate parts makes sense because we use an `EntityInfo` component in the other repos but without the menu and the key facts.  By having a component that just displays the entity name and status, it would allow us to reuse the component in multiple places and have a consistent look as users jump from application to application.  (I'm stepping out on a limb here because I'm assuming the UX folks haven't asked from the `EntityInfo` header to vary between applications.)

Reply:
Good points. I'm glad you're thinking clearly, haha.

I'll check on what blocks might be common between our 3 UIs, and if it makes sense then I'll create sub-components for them here. Later, we can reuse those sub-components in the other UIs.

Reply:
I identified that only the right hand column ("tombstone") data might be common between Create/Edit/Filings UI, so I refactored that into its own sub-component. See third commit of this PR.